openmdao/drivers/doe_driver.py

```diff
@@ -240,29 +240,14 @@ def _parallel_generator(self, design_vars, model=None):
             if i % size == color:
                 yield case
 
-    def add_recorder(self, recorder):
-        """
-        Add a recorder to the driver.
-
-        Parameters
-        ----------
-        recorder : CaseRecorder
-           A recorder instance.
-        """
-        # keep track of recorders so we can flag them as parallel
-        # if we end up running in parallel
-        self._recorders.append(recorder)
-
-        super().add_recorder(recorder)
-
```

Comment:
should be able to get rid of the `_recorders` attribute as well...

Reply:
Agreed. Removed

Reply:
I notice that the attribute is still included in the class docstring...  I thought linting would catch this but maybe it only detects undocumented attributes... anyway, I think we need to delete this in line 27:
```
    _recorders : list
        List of case recorders that have been added to this driver.
```

Reply:
Ah, another good catch. Removed